### PR TITLE
cephfs-shell: excute cmd 'rmdir_helper' reported error

### PR DESCRIPTION
**Pull request by tengjie5**
cephfs-shell: excute cmd 'rmdir_helper' reported error
Fixes: https://tracker.ceph.com/issues/66735
Signed-off-by: teng jie <tengjie5@asiainfo.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by joscollin — APPROVED** [review]

File: src/tools/cephfs/shell/cephfs-shell [hunk 1]
```diff
@@ -1014,9 +1014,9 @@ class CephFSShell(Cmd):
 
     @with_argparser(rmdir_parser)
     def do_rmdir(self, args):
-        self.do_rmdir_helper(args)
+        self.rmdir_helper(args)
 
-    def do_rmdir_helper(self, args):
+    def rmdir_helper(self, args):
```
Comment (batrick) [inline src/tools/cephfs/shell/cephfs-shell:1019]:
```suggestion
    def _rmdir(self, args):
```

would be better, no?

**Review by batrick — CHANGES_REQUESTED** [review]

**Comment by vshankar** [comment]
@tengjie5 ping

File: src/tools/cephfs/shell/cephfs-shell (hunk 1 above)
Comment (tengjie5) [reply src/tools/cephfs/shell/cephfs-shell]: @batrick please review it again，thanks.

**Review by batrick — APPROVED** [review]

**Review by rishabh-d-dave — APPROVED** [review]

**Comment by batrick** [comment]
This PR is under test in [https://tracker.ceph.com/issues/66822](https://tracker.ceph.com/issues/66822).

**Comment by batrick** [comment]
https://pulpito.ceph.com/pdonnell-2024-07-04_01:31:57-fs-wip-pdonnell-testing-20240703.143006-debug-distro-default-smithi/7786631/

LGTM

**Pull request merged**